vng_api_common/utils.py

```diff
@@ -110,3 +110,22 @@ def request_object_attribute(url: str, attribute: str, resource: Union[str, None
         logger.warning("%s was retrieved from %s with the %s: %s", attribute, url, exc.__class__.__name__, exc)
         result = ''
     return result
+
+
+def generate_unique_identification(instance: models.Model, date_field_name: str):
+    model_name = instance._meta.model_name.upper()
+    year = getattr(instance, date_field_name).year
+    prefix = f'{model_name}-{year}'
+
+    pattern = prefix + r'-\d{10}'
+
+    issued_ids_for_year = instance.__class__.objects.filter(identificatie__regex=pattern)
```

Comment:
Django marks the first defined manager as 'default' so you don't have to rely on the manager being named `objects`

```suggestion
    issued_ids_for_year = instance.__class__._default_manager.filter(identificatie__regex=pattern)
```